desktop/core/src/desktop/js/parse/jison/hplsqlStatementsParser.jison

```diff
@@ -0,0 +1,201 @@
+// Licensed to Cloudera, Inc. under one
```

Comment:
Is it ready or should be split in a separate PR?

Reply:
Not ready, I also think better split in a separate PR.
Okay I will do that.